diff --git a/addons/gdUnit4/src/doubler/CallableDoubler.gd b/addons/gdUnit4/src/doubler/CallableDoubler.gd
index 14a5947..88e598e 100644
--- a/addons/gdUnit4/src/doubler/CallableDoubler.gd
+++ b/addons/gdUnit4/src/doubler/CallableDoubler.gd
@@ -81,16 +81,7 @@ func bindv(caller_args: Array) -> Callable:
 
 
 @warning_ignore("untyped_declaration", "native_method_override", "unused_parameter")
-func call(arg0=null,
-	arg1=GdObjects.TYPE_VARARG_PLACEHOLDER_VALUE,
-	arg2=GdObjects.TYPE_VARARG_PLACEHOLDER_VALUE,
-	arg3=GdObjects.TYPE_VARARG_PLACEHOLDER_VALUE,
-	arg4=GdObjects.TYPE_VARARG_PLACEHOLDER_VALUE,
-	arg5=GdObjects.TYPE_VARARG_PLACEHOLDER_VALUE,
-	arg6=GdObjects.TYPE_VARARG_PLACEHOLDER_VALUE,
-	arg7=GdObjects.TYPE_VARARG_PLACEHOLDER_VALUE,
-	arg8=GdObjects.TYPE_VARARG_PLACEHOLDER_VALUE,
-	arg9=GdObjects.TYPE_VARARG_PLACEHOLDER_VALUE) -> Variant:
+func call(arg0=null, ...args) -> Variant:
 
 	# This is a placeholder function signanture without any functionallity!
 	# It is used by the function doubler to double function signature of Callable:call()
